src/models/models.jl

```diff
@@ -0,0 +1,185 @@
+export PolarForm, get, bounds, powerflow
```

Comment:
Implement all the abstractions needed to implement new formulations (polar, rectangular) in a modular way
We should discuss here what to export exactly